Makefile

```diff
@@ -197,7 +203,7 @@ proto-all: proto-format proto-lint proto-gen format
 
 proto-gen:
 	@echo "Generating Protobuf files"
-	@$(protoImage) sh ./scripts/protocgen.sh
+	sh ./scripts/protocgen.sh
```

Comment:
`protoImage` was reverted away here but we are using it for some other proto commands. Do we want it or no?